charts/consul/test/terraform/aks/variables.tf

```diff
@@ -16,10 +16,18 @@ variable "client_secret" {
 variable "cluster_count" {
   default     = 1
   description = "The number of Kubernetes clusters to create."
+
+  // We currently cannot support more than 2 cluster
+  // because setting up peering is more complicated if cluster count is
+  // more than two.
+  validation {
+    condition     = var.cluster_count < 3 && var.cluster_count > 0
+    error_message = "The cluster_count value must be 1 or 2."
+  }
```

Comment:
Ooh, I might need to steal something like this for the EKS branch, since it only supports 2 clusters right now.

Reply:
Haha great minds! I just left a comment on your PR saying that 😄